packages/components/atoms/f-button/stories/RouterButton.stories.js

```diff
@@ -0,0 +1,82 @@
+import Vue from 'vue';
+import Router from 'vue-router';
+
+import {
+    withKnobs,
+    select,
+    boolean,
+    text
```

Comment:
text can be removed as not in use anymore

Reply:
done